src/components/Reserves.js

```diff
@@ -106,12 +128,16 @@ class Reserves extends React.Component {
   renderResultsFirstMenu = (filters) => {
     const { filterPaneIsVisible } = this.state;
     const filterCount = filters.string !== '' ? filters.string.split(',').length : 0;
-    const hideOrShowMessageId = filterPaneIsVisible ?
-      'stripes-smart-components.hideSearchPane' : 'stripes-smart-components.showSearchPane';
+    const hideOrShowMessageId = filterPaneIsVisible
+      ? 'stripes-smart-components.hideSearchPane'
+      : 'stripes-smart-components.showSearchPane';
 
     return (
       <PaneMenu>
-        <FormattedMessage id="stripes-smart-components.numberOfFilters" values={{ count: filterCount }}>
+        <FormattedMessage
```

Comment:
I do wish that when adding new functionality you would not also make personal-preference whitespace chages like this. It makes it much more complicated to review the code.

(I'm not saying you should change it back, just that doing this kind of thing in passing should be avoided.)

Reply:
Okay, I will take this into account, thanks.

Reply:
Thanks, appreciated!